include/shared_memory_cache_helper.cuh

```diff
@@ -210,6 +210,34 @@ namespace quda
       return load_detail(tid.x, tid.y, z);
     }
 
+    /**
+       @brief Save a vector to the shared memory cache but with cutomized index and leading dimension
+       @param[in] index The customized index
+       @param[in] ld The leading dimension
+     */
+    __device__ inline void save_idx(int index, int ld, const T &a)
```

Comment:
Wondering if we should change the ordering here to something like the below to match the overload for `save` which has optionals thread coordinates after the parameter `a`?
```c++
__device__ inline void save_idx(const T &a, int index, int ld)
```

Reply:
https://github.com/lattice/quda/pull/1061/commits/d776b593adac861010d314b14523ba261ada3c14 has made this change unnecessary.